### DISCUSSION TITLE
is it possible to remember the pos which was replicated?

### DISCUSSION THREAD

**Discussion by sixwaaaay** [julien-duponchelle/python-mysql-replication#589] [Q&A]
I don't want to dump from the beginning on each running

**Comment by sean-k1** [comment]
I dont understand exactly what you said.
Do You want to stream specific bin log pos?

**Reply by sixwaaaay** [reply]
yes.
Each time I re-run the program, it re-processes these rows instead of ignoring the rows that have already been processed.

**Reply by sean-k1** [reply]
@sixwaaaay 
set binlogStream like this.

eg)
```
    stream = BinLogStreamReader(
        connection_settings=MYSQL_SETTINGS, server_id=3, blocking=True, log_pos=644, log_file='binlog.000013', resume_stream=True
    )

```
